src/runner.ts

```diff
@@ -401,4 +402,14 @@ export class StaticAnalysisParserRunner {
             return severityOrder[nextVuln.severity] - severityOrder[currentVuln.severity];
         });
     }
+
+    private extractRelativePath(basePath: string, targetPath: string): string {
+        const normalizedBase = pt.normalize(basePath).replace(/\\/g, '/');
+        const normalizedTarget = pt.normalize(targetPath).replace(/\\/g, '/');
+        const prefix = `${normalizedBase}/`;
```

Comment:
It should be: `const prefix = normalizedBase.endsWith('/') ? normalizedBase : normalizedBase + '/';`